index.html

```diff
@@ -946,14 +864,143 @@ <h4>publicKeyBase58</h4>
                 <a href="https://w3id.org/security/#publicKeyBase58">security-vocab</a>
               </td>
               <td>
-                <a href="https://www.w3.org/ns/did/v1">DID Core context</a>
+                <a href="https://www.w3.org/ns/did/v1">DID Core</a>
               </td>
               <td>
                 <a href="cddl/publicKeyBase58.cddl" target="_blank">publicKeyBase58.cddl</a>
               </td>
             </tr>
           </tbody>
         </table>
+      </section>
+
+        <section>
+          <h4>publicKeyHex</h4>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+                <th>CDDL</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a href="https://w3id.org/security#publicKeyHex">security-vocab</a>
+                </td>
+                <td>
+                  <a href="https://w3id.org/security/v3-unstable">security-vocab/v3-unstable</a>
+                </td>
+                <td>
+                </td>
+                <td>
+                  <a href="cddl/publicKeyHex.cddl" target="_blank">publicKeyHex.cddl</a>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <pre class="example" title="Example of publicKeyHex property">
+  {
+    "@context":[
+      "https://www.w3.org/ns/did/v1",
+      "https://identity.foundation/EcdsaSecp256k1RecoverySignature2020#"
+    ],
+    "id":"did:example:123",
+    "publicKey":[{
+      "id": "did:example:123#vm-2",
+      "controller": "did:example:123",
+      "type": "EcdsaSecp256k1RecoveryMethod2020",
+      "publicKeyHex": "027560af3387d375e3342a6968179ef3c6d04f5d33b2b611cf326d4708badd7770"
+    }]
+  }
+          </pre>
+        </section>
+
+        <section>
+          <h4>blockchainAccountId</h4>
+          <div class="issue" data-number="165"></div>
+          <table class="simple" style="width: 100%;">
+            <thead>
+              <tr>
+                <th>Normative Definition</th>
+                <th>JSON-LD</th>
+                <th>CDDL</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>
+                  <a href="https://w3id.org/security#blockchainAccountId">security-vocab</a>
+                </td>
+                <td>
+                  <a href="https://www.w3.org/ns/did/v1">DID Core</a>
+                </td>
+                <td>
+                  <span class="issue">CDDL definition pending</span>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+          <pre class="example" title="Example of blockchainAccountId property">
+  {
+    "@context":[
+      "https://www.w3.org/ns/did/v1",
```

Comment:
```suggestion
      "https://www.w3.org/ns/did/v1",
      "https://w3id.org/security/v3-unstable",
```

Reply:
again, this was added to did core intentionally, the public ledger did community does not appreciate being required to add JSON-LD extensions to represent basic concepts like an address :)